db/version_edit_handler.h

```diff
@@ -36,7 +36,7 @@ class VersionEditHandler {
       bool read_only,
       const std::vector<ColumnFamilyDescriptor>& column_families,
       VersionSet* version_set, bool track_missing_files,
-      bool ignore_missing_files);
+      bool ignore_missing_files, const std::shared_ptr<IOTracer>& io_tracer);
```

Comment:
Since io_tracer is a shared_ptr type, is it necessary to use reference?

Reply:
I used reference to avoid copy as this pointer will not be deleted till DBImpl is destroyed or someone call DB::EndIOTrace().